autoblocks/_impl/api/models.py

```diff
@@ -201,3 +201,35 @@ class Dataset:
     schema_version: str
     revision_id: str
     items: List[DatasetItem]
+
+
+@dataclass
+class AutoblocksTestRun:
+    id: str
+
+
+@dataclass
+class AutoblocksTestCaseResultId:
+    id: str
+
+
+@dataclass
+class Evaluation:
+    evaluatorId: str
+    score: float
+    passed: bool
+    metadata: Dict[str, Any]
+    threshold: Optional[Dict[str, Optional[float]]] = None
```

Comment:
are we returning assertions from the public api? If not we should and also set that in both js sdk and python sdk